templates/security/login_user.html

```diff
@@ -16,7 +16,7 @@ <h1>Login</h1>
                 {{ render_field(login_user_form.submit, class="btn btn-primary") }}
             </form>
             {% if security.registerable %}
-            <p>Not yet signed up? Please <a href="{{ url_for('security.register') }}">register for an account</a>.</p>
```

Comment:
why do we need a new registration function?

Reply:
i cant find a way to get recaptcha in the form.

Reply:
i dont think we need the recaptcha